problemsets-mat223/linearalbegra.tex

```diff
@@ -5377,13 +5377,56 @@ \section*{Eigenvectors}
 	\question
 	Define $F=\mat{1&1\\0&1}$.
 	\begin{parts}
-		\item Is $F$ diagonalizable?  Why or why not?
+		\item Is $F$ diagonalizable? Why or why not?
+			\begin{solution}
+				No.
+
+				$\chr(F)=(1-\lambda)^2$, and so the only eigenvalue of $F$ is $1$.
+				$F$ is diagonalizable if and only if there is a basis of $\R^2$
```

Comment:
How about we start by computing all eigen vectors, then notice that they don't form a basis, then conclude it's not diagonalizable.